dockers/docker-orchagent/config.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash -e
+
+mkdir -p /etc/swss/config.d/
+
+sonic-cfggen -m /etc/sonic/minigraph.xml -t /usr/share/sonic/templates/ipinip.json.j2 > /etc/swss/config.d/ipinip.json
```

Comment:
I think we need to support the case that ErspanDestinationIpv4 is not defined in minigraph as well. Current implementation will generate an illegal json therefore crashes docker-orchagent.